Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/Pass.h

```diff
@@ -479,6 +485,9 @@ namespace AZ
             // For image attachment preview
             AZStd::weak_ptr<ImageAttachmentCopy> m_attachmentCopy;
 
+            //! Optional data used during pass initialization
+            AZStd::shared_ptr<PassData> m_passData = nullptr;
```

Comment:
Is this data a copy or just a reference from a pass template? Should it be const?